.readthedocs.yml

```diff
@@ -8,10 +8,5 @@ version: 2
 conda:
   environment: documentation/conda.yml
 
-python:
-  install:
-    - method: pip
-      path: .
```

Comment:
By doing this, we lose the ability to see seperate logs in rtd for our build vs that of dependencies.

I preferred the first iteration of this patch which touched only the makefile.